qhub/template/stages/07-kubernetes-services/modules/kubernetes/services/kbatch/main.tf

```diff
@@ -27,7 +27,7 @@ resource "helm_release" "kbatch" {
         }
       }
       image = {
-        tag = "0.3.1"
+        tag = "0.4.0"
```

Comment:
How about putting these versions in [`../kbatch/variables.tf`](https://github.com/Quansight/qhub/blob/7ca7f2f61765036a1095505f56f603356387e9a6/qhub/template/stages/07-kubernetes-services/modules/kubernetes/services/kbatch/variables.tf)? So that we have to change it at one place only.

Reply:
Yeah, that's definitely a better approach! Thanks :)